ext/tokenizer/tokenizer.stub.php

```diff
@@ -2,9 +2,9 @@
 
 /** @generate-function-entries */
 
-function token_get_all(string $source, int $flags = 0): array {}
+function token_get_all(string $code, int $flags = 0): array {}
 
-function token_name(int $token): string {}
+function token_name(int $token_id): string {}
```

Comment:
I would be ok with `$id` too. E.g. `filter_id()` uses `$name` :D But I'm ok with using `$token_id` here, or changing `$name` to `$filter_name` in `filter_id()`